src/components/textfieldButtons/js/TextfieldButtons.js

```diff
@@ -0,0 +1,166 @@
+/*
+Copyright 2013 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+var fluid_2_0_0 = fluid_2_0_0 || {};
+
+(function ($, fluid) {
+    "use strict";
+
+    /********************
+     * Textfield Buttons *
+     ********************/
+
+    fluid.defaults("fluid.textfieldButtons", {
+        gradeNames: ["fluid.viewComponent"],
+        components: {
+            textfield: {
+                type: "fluid.textfieldButtons.textfield",
+                container: "{textfieldButtons}.dom.textfield",
+                options: {
+                    model: "{textfieldButtons}.model",
+                    range: "{textfieldButtons}.options.range"
+                }
+            },
+            increaseButton: {
+                type: "fluid.stepper.button",
+                container: "{textfieldButtons}.dom.increaseButton",
+                options: {
+                    model: "{fluid.textfieldButtons}.model",
```

Comment:
I am trying do that, but I don't how to have 2 different targets with the same record. The targets are {that > increaseButton}.options and {that > decreaseButton}.options.